examples/fabric-bridge-app/linux/main.cpp

```diff
@@ -129,26 +129,45 @@ void AdministratorCommissioningCommandHandler::InvokeCommand(HandlerContext & ha
     }
 
     handlerContext.SetCommandHandled();
-    Status status = Status::Success;
+
+    uint16_t commissioningTimeout;
+    uint16_t discriminator;
+    uint32_t iterations;
+    chip::Optional<chip::ByteSpan> salt;
+    chip::Optional<chip::ByteSpan> verifier;
+    Device * device = nullptr;
+    Status status   = Status::Failure;
+
+    Commands::OpenCommissioningWindow::DecodableType commandData;
+    CHIP_ERROR tlvError = DataModel::Decode(handlerContext.mPayload, commandData);
+    SuccessOrExit(tlvError);
+
+    commissioningTimeout = commandData.commissioningTimeout;
```

Comment:
why do you separate declaration of variables from assignment? I assume because of successOrExit... can we avoid that pattern?

Do we need separate variables? why not use commandData.discriminator and such directly and not have separate variables?

Reply:
Fixed in https://github.com/project-chip/connectedhomeip/pull/34209/commits/5d4ce206b24d3a3a4201e6d1cdc816ca6e7f8b15